docs/source/ucx-intro.rst

```diff
@@ -0,0 +1,91 @@
+Send/Recv Internals
```

Comment:
Why do we have this a second time? Seems the same content as of https://github.com/rapidsai/ucx-py/pull/560/files#diff-6d59a7bf7a6266fb37c46a99dcc04b78 .

Reply:
It's a mistake. I'll clean it up now